account_invoice_date_maturity/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Invoice Date Maturity",
+    "summary": """
+        Adds date_maturity field with Journal Entries action
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Account",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": [
+        "account",
+    ],
+    "data": [
+        "views/account_move_views.xml",
```

Comment:
Action redefined in this file belongs to `account.move.line` model, so this file should be renamed.

Reply:
Oops I was wrong, such action belongs to `account.move`. Forget it.